packages/components/src/List/ListComposition/Manager/ListManager.component.js

```diff
@@ -1,19 +1,28 @@
-import React, { useState } from 'react';
+import React, { useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
 import { translate } from 'react-i18next';
 
 import { ListContext } from '../context';
 import getDefaultT from '../../../translate';
 import I18N_DOMAIN_COMPONENTS from '../../../constants';
+import { sortCollection } from './helpers';
 
 function Manager(props) {
 	const [displayMode, setDisplayMode] = useState();
+	const [sortParams, setSortParams] = useState({});
+
+	let collection = props.collection;
+
+	// Sort items
+	collection = useMemo(() => sortCollection(collection, sortParams), [collection, sortParams]);
```

Comment:
May we use the props here & remove the let at line 14 ?
```suggestion
	const collection = useMemo(() => sortCollection(props.collection, sortParams), [collection, sortParams]);
```

Reply:
I used `let` because `collection` will be overwritten with the filter output (in an other PR), it was this way in the previous main pull request. I can change it back but we will have to change it back back later so ... 😶

Reply:
Allright :)